deploy/charts/trust-manager/templates/webhook.yaml

```
@@ -1,3 +1,52 @@
+{{- if and .Values.app.webhook.tls.helmCert.enabled .Values.app.webhook.tls.approverPolicy.enabled -}}
+{{- fail "cannot set .app.webhook.tls.helmCert.enabled and .Values.app.webhook.tls.approverPolicy.enabled" }}
+{{- end -}}
+
+{{- /*
+$ca is always generated here even if .Values.app.webhook.tls.helmCert.enabled is false because we need it to be
+visible in the scope for the ValidatingWebhookConfiguration below.
+
+genCA has two args - first is cert commonName and second is validity in days (9125 = ~25 years)
```

```suggestion
genCA has two args - first is cert CN and second is validity in days (9125 = ~25 years)
```
Nit: To be consistent in the terminology.

---

I've changed to "common name (CN)" so the abbreviation below is defined - good suggestion, thanks 👍